numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -55,6 +58,44 @@
 _py_decref.argtypes = [ctypes.py_object]
 _py_incref.argtypes = [ctypes.py_object]
 
+pynvjitlink_import_err = None
+try:
+    from pynvjitlink.api import NvJitLinker, NvJitLinkError
+except ImportError as err:
+    pynvjitlink_import_err = err
+
+
+def _readenv(name, ctor, default):
+    value = os.environ.get(name)
+    if value is None:
+        return default() if callable(default) else default
+    try:
+        return ctor(value)
+    except Exception:
+        warnings.warn(
+            f"Environment variable '{name}' is defined but its associated "
+            f"value '{value}' could not be parsed.\n"
+            "The parse failed with exception:\n"
+            f"{traceback.format_exc()}",
+            RuntimeWarning
+        )
+        return default
+
+
+config.ENABLE_PYNVJITLINK = False
+if _readenv("ENABLE_PYNVJITLINK", bool, False):
+    config.ENABLE_PYNVJITLINK = True
```

Comment:
Doing this here causes the config variable to be overwritten if the user set it prior to importing `numba.cuda`. e.g.:

```python
In [1]: from numba.core import config

In [2]: config.ENABLE_PYNVJITLINK = True

In [3]: from numba import cuda

In [4]: config.ENABLE_PYNVJITLINK
Out[4]: False
```